src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -66,27 +66,22 @@ object DepositHandler {
       }
     settings
       .tempDir.listFiles().toSeq
```

Comment:
Not sure, but do you still need the `.toSeq` here?

Reply:
I guess not